exchange/exchangetest/passthrough.json

```diff
@@ -0,0 +1,122 @@
+                    "site": {
+                        "page": "test.somepage.com"
+                    },
+                    "imp": [{
+                        "id": "my-imp-id",
+                        "video": {
+                            "mimes": ["video/mp4"]
+                        },
+                        "ext": {
+                            "bidder": {
+                                "placementId": 1
+                            },
+                            "prebid": {
+                                "passthrough": {
+                                    "passthrough_val": 20
+                                }
+                            }
+                        }
+                    }],
+                    "ext": {
+                        "prebid": {
+                            "passthrough": {"passthrough_val":20}
+                        }
+                    }
+                }
+            },
+            "mockResponse": {
+                "pbsSeatBids": [{
+                    "pbsBids": [{
+                        "ortbBid": {
+                            "id": "apn-bid",
+                            "impid": "my-imp-id",
+                            "price": 0.3,
+                            "w": 200,
+                            "h": 250,
+                            "crid": "creative-1",
+                            "ext": {
+                                "someField": "someValue",
+                                "origbidcpm": 0.3,
+                                "prebid": {
+                                    "willBeOverwritten": "by core logic",
+                                    "passthrough": {
+                                        "passthrough_val": 20
+                                    }
+                                }
+                            }
+                        },
+                        "bidType": "video"
+                    }],
+                    "seat": "appnexus"
+                }]
+            }
+        }
+    },
+    "response": {
+        "bids": {
+            "id": "some-request-id",
+            "seatbid": [{
+                "seat": "appnexus",
+                "bid": [{
+                    "id": "apn-bid",
+                    "impid": "my-imp-id",
+                    "price": 0.3,
+                    "w": 200,
+                    "h": 250,
+                    "crid": "creative-1",
+                    "ext": {
+                        "someField": "someValue",
+                        "origbidcpm": 0.3,
+                        "prebid": {
+                            "type": "video",
+                            "passthrough": {
+                                "passthrough_val": 20
+                            }
+                        }
+                    }
+                }]
+            }]
+        },
+        "ext": {
+            "prebid": {
+                "passthrough": {"passthrough_val":20}
+            }
+        }
```

Comment:
The test framework doesn't appear to be attempting to validate this portion of the response. If I remove this `ext` block the test still passes.

Reply:
This still appears to be a problem but I think my suggested fix here should fix it: https://github.com/prebid/prebid-server/pull/2308#discussion_r930549815